src/interface/iface.rs

```diff
@@ -119,6 +119,14 @@ pub struct ValencyIface {
     pub multiple: bool,
 }
 
+impl ValencyIface {
+    pub fn any(req: Req) -> Self {
```

Comment:
I am not sure I understand what is the purpose of this function. It also seems to be named incorrectly (it does something different to its name).

Reply:
ok, maybe I misunderstand the purpose of this function, I will delete it.